portal_pages/templates/portal_pages/blog_index_page.html

```diff
@@ -29,6 +29,8 @@ <h1 class="cushion">{{ self.title }}</h1>
                         {% endif %}
 
                     </div>
+                    <br />
+                    <a class="button" href="{% routablepageurl self "submit" %}">Submit Your Own Blog Page</a>
```

Comment:
I would call this "Blog Post" or "Blog Entry" rather than "Blog Page"...for some reason "Blog Page" sounds odd...